src/dbnode/persist/fs/retriever_test.go

```diff
@@ -299,6 +299,10 @@ func testBlockRetrieverHighConcurrentSeeks(t *testing.T, shouldCacheShardIndices
 			require.True(t, ok, fmt.Sprintf("expected %s to be retrieved, but it was not", id))
 
 			expectedTags := ident.NewTags(testTagsFromTestID(id)...)
+			if !tags.Equal(expectedTags) {
+				fmt.Println("expected: ", len(expectedTags.Values()))
+				fmt.Println("actual: ", len(tags.Values()))
```

Comment:
Can you make this a message instead of using fmt.Prints?

i.e.
```go
require.True(t, tags.Equal(expectedTags), 
	fmt.Sprintf("expectedNumTags=%d, actualNumTags=%d", len(expectedTags.Values()), len(tags.Values()))
```